src/dataconnect/graphqlError.ts

```diff
@@ -17,6 +17,8 @@ export function prettifyWithWorkaround(errs: GraphqlError[]): string {
   const table = new Table({
     head: ["Issue", "Workaround", "Reason"],
     style: { head: ["yellow"] },
+    colWidths: [75, 75, 75],
```

Comment:
@joehan not sure if there's a better way of determining a default column size. Is there a good way to get the terminal window size programmatically?